### DISCUSSION TITLE
Workflow safety and compile times

### DISCUSSION THREAD

**Discussion by AleDema** [nxrighthere/UnrealCLR#127] [General]
Hi I really appreciate what you're creating, I have a couple questions on the improvements unrealCLR brings to the table. If I use only BPs and C# scripts will that improve compile times and prevent the engine from crashing to desktop in case I mess up something?

**Comment by nxrighthere** [comment]
> If I use only BPs and C# scripts will that improve compile times and prevent the engine from crashing to desktop in case I mess up something?

Yes, safe code execution is one of the goals of this project. All exposed functionality is tested under different conditions, but mostly on Windows.

**Reply by AleDema** [reply]
Awesome, what about compile times? Will they be faster than C++?

**Reply by nxrighthere** [reply]
Yes, compile times will be faster in a complex codebase, especially cold start.
